packages/devextreme/js/__internal/ui/chat/chat.ts

```diff
@@ -32,6 +32,9 @@ const TEXTEDITOR_INPUT_CLASS = 'dx-texteditor-input';
 type Properties = ChatProperties & {
   title: string;
   showDayHeaders: boolean;
+  showAvatar: boolean;
+  showUsername: boolean;
+  showMessageTimestamp: boolean;
```

Comment:
It should be removed after .d.ts is updated https://github.com/DevExpress/DevExtreme/pull/28219

Reply:
d606e6df0f52b1d9a29507189f1525912d86d4ca